_episodes/06-script.md

```diff
@@ -147,10 +147,16 @@ ATOM     13  H           1      -1.183   0.500  -1.412  1.00  0.00
 > we surround `$1` with double-quotes.
 {: .callout}
 
-We still need to edit `middle.sh` each time we want to adjust the range of lines,
-though.
-Let's fix that by using the special variables `$2` and `$3` for the
-number of lines to be passed to `head` and `tail` respectively:
+Currently, we need to edit `middle.sh` each time we want to adjust the range of 
+lines that is returned. 
+Let's fix that by configuring our script to instead use three command-line arguments. 
+After the first command-line argument ($1), each additional argument that we 
+provide will be assigned a special variable name of increasing interger values 
+such that $1, $2, $3, etc. refer to the first, second, third, etc. command-line 
```

Comment:
These two mid-sentence et ceteras don't read well. I think we can safely remove them both, please.

Reply:
I have removed 'etc.' in the third commit